students/12th/koJaeWon/login.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>letsKoJae_star</title>
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/login.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body>
+    <div class="allWrap">
+      <div class="wrap">
+        <!-- instaLogo / id / password -->
+        <div class="logInPage">
+          <img alt="instaLogo" src="img/logo_text.png" class="logoImg">
+          <div class="form">
+            <div class="userIn">
+              <input type="text" class="log" placeholder="전화번호, 사용자 이름 또는 이메일">
+            </div>
+            <div class="userIn">
+              <input type="text" class="pass" placeholder="비밀번호">
+            </div>
+            <div class="btn">
+              <button class="logBtn">로그인</button>
+            </div>
+            <div class="forGot">
```

Comment:
```suggestion
            <div class="forgot">
```
카멜케이스는 단어단위로 끊어서 적용해주세요! for got으로 읽힙니다!

Reply:
forgot으로 변경했습니다!